web/templates/user_group_settings/user_group_settings_overlay.hbs

```diff
@@ -32,20 +32,48 @@
                     </span>
                 </div>
                 <div class="no-groups-to-show">
-                    <div class="your_groups_tab_empty_text">
+                    <div class="active_and_deactivated_your_groups_tab_empty_text">
                         <span class="settings-empty-option-text">
                             {{t 'You are not a member of any user groups.'}}
                             <a href="#groups/all">{{t 'View all user groups'}}</a>
                         </span>
                     </div>
-                    <div class="all_groups_tab_empty_text">
+                    <div class="active_your_groups_tab_empty_text">
+                        <span class="settings-empty-option-text">
+                            {{t 'You are not a member of any active user groups.'}}
+                            <a href="#groups/all">{{t 'View all user groups'}}</a>
+                        </span>
+                    </div>
+                    <div class="deactivated_your_groups_tab_empty_text">
+                        <span class="settings-empty-option-text">
+                            {{t 'You are not a member of any deactivated user groups.'}}
+                            <a href="#groups/all">{{t 'View all user groups'}}</a>
+                        </span>
+                    </div>
+                    <div class="active_and_deactivated_all_groups_tab_empty_text">
                         <span class="settings-empty-option-text">
                             {{t 'There are no user groups you can view in this organization.'}}
                             {{#if can_create_user_groups}}
                                 <a href="#groups/new">{{t 'Create a user group'}}</a>
                             {{/if}}
                         </span>
                     </div>
+                    <div class="active_all_groups_tab_empty_text">
+                        <span class="settings-empty-option-text">
+                            {{t 'There are no active user groups you can view in this organization.'}}
+                            {{#if can_create_user_groups}}
+                                <a href="#groups/new">{{t 'Create a user group'}}</a>
+                            {{/if}}
+                        </span>
+                    </div>
+                    <div class="deactivated_all_groups_tab_empty_text">
+                        <span class="settings-empty-option-text">
+                            {{t 'There are no deactivated user groups you can view in this organization.'}}
+                            {{#if can_create_user_groups}}
+                                <a href="#groups/new">{{t 'Create a user group'}}</a>
+                            {{/if}}
+                        </span>
+                    </div>
```

Comment:
Having all of these in the DOM is proving rather messy. I think you should just extract this block to a new `web/templates/user_group_settings/user_group_settings_empty_notice.hbs`, and have the JS code replace the contents of `no-groups-to-show` (or empty it) on rerender, rather than having 6 things that are always here and toggling which are visible.

I think doing so will save you needing to name a half-dozen classes like `deactivated_all_groups_tab_empty_text`, which should be using `-`s rather than `_`s, and are too long, but we should get rid of them anyway.